root/genre/GenreListPage.js

```diff
@@ -21,13 +21,12 @@ const GenreListPage = ({genres}: PropsT) => (
     <div id="content">
       <h1>{l('Genre List')}</h1>
       <p>
-        {l(`These are all the tags that will be understood 
-            as genres by the tag system.`)}
+        {l('These are all the genres currently available in MusicBrainz.')}
```

Comment:
The previous sentence explained the new link between genres and tags, but the new sentence doesn’t give much information and there is no such sentence on the top of other lists. Please remove it if unneeded, otherwise update the instrument list page (`root/instrument/List.js`) too.

Reply:
Left this but also added a second line explaining how to use them. How does that sound?

Reply:
Better! A link to the documentation `/doc/Genre` would be nice too.